src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -39,6 +40,11 @@
 MIN_DAYS = 7
 
 
+class MethodString(StrEnum):
```

Comment:
Hmm i'd be a little confused by this name; `MethodString` as i'd expect `MethodString.CREATE` to return `POST` - thinking it'd return the expected HTTP Method.

Maybe something to describe what the method is for like:
```suggestion
class SeerMethod(StrEnum):
```
by prefixing the name with seer, i can easily grok that "hey, these are the methods that seer specifically is expecting us to use"

Reply:
lol I had a hard time coming up with a good name for this 😅